### PR TITLE
slight fixes

### PR DESCRIPTION
**Pull request by Razor421**
did the following
turned latin in:
contacts
FAQ
and thats it pretty much
also doing 404 page investigation

### CODE REVIEW

File: Contact_Us/contact.html [hunk 1]
```diff
@@ -68,8 +68,11 @@
     <div class="contactDetails">
       <span>
         <h2><i class="fas fa-paper-plane"></i> Contact Us</h2>
-        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Eius soluta eveniet delectus nemo, reiciendis
-          repellendus.</p>
+        <p>you can contact our team using our
```
Comment (0xMukesh) [inline Contact_Us/contact.html]: Make the "Y" capital 😅

File: Contact_Us/contact.html [hunk 2]
```diff
@@ -68,8 +68,11 @@
...
+          <a href="mailto:bteasean@gmail.com">official email
+            <a href="https://discord.gg/thA62UNzRy">or our discord server.</a>
```
Comment (0xMukesh) [inline Contact_Us/contact.html]: Add the `anchor tag` only to the term `discord` not entirely to `or our discord server`

File: FAQ/index.html [hunk 3]
```diff
@@ -82,14 +82,14 @@ <h2>FAQ</h2>
                                 <div class="mb-0">
                                     <h5 class="faq-title" data-toggle="collapse" data-target="#faqCollapse-1"
                                         data-aria-expanded="true" data-aria-controls="faqCollapse-1">
-                                        <span class="badge">1</span>Lorem ipsum dolor sit amet.
+                                        <span class="badge">1</span>how do i join the build team?
```
Comment (0xMukesh) [inline FAQ/index.html]: Make "H" capital

Comment (0xMukesh) [reply FAQ/index.html]: 😅 Take care of the typography

**Review by 0xMukesh — COMMENTED** [review]
@Razor421 Minor fixes needed

File: FAQ/index.html (hunk 3 above)
Comment (Razor421) [reply FAQ/index.html]: oops

File: Contact_Us/contact.html (hunk 2 above)
Comment (Razor421) [reply Contact_Us/contact.html]: yessir!

File: Contact_Us/contact.html (hunk 1 above)
Comment (Razor421) [reply Contact_Us/contact.html]: yes sir!

**Pull request closed without merging**